starfish/pipeline/registration/fourier_shift.py

```diff
@@ -1,4 +1,8 @@
+from typing import Union
+
 from starfish.constants import Indices
```

Comment:
These changes also duplicate another pending PR that were needed for this work. Ignore this module.